hack/make-rules/verify.sh

```diff
@@ -35,6 +35,7 @@ EXCLUDED_PATTERNS=(
   "verify-linkcheck.sh"          # runs in separate Jenkins job once per day due to high network usage
   "verify-*-dockerized.sh"       # Don't run any scripts that intended to be run dockerized
   "verify-govet-levee.sh"        # Do not run levee analysis by default while KEP-1933 implementation is in alpha.
+  "verify-licenses.sh"           # runs in a separate job
```

Comment:
`runs in a separate job to monitor availability of the dependencies periodically`

Reply:
@dims, acknowledged. updated the comment to the text above.
Thank you for suggesting a clearer message.

removing hold, since PR #109299 is merged.
/unhold